### PR TITLE
Modify redundant code in modal

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
modify modal

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 1]
```diff
@@ -249,21 +249,29 @@ const Configuration = () => {
...
+                })
+              : update({
+                  modalShow: false,
+                });
+          }}
           width={720}
-          title={i18n.t('msp:accessKey details')}
-          visible={keyDetailShow}
+          title={keyDetailShow ? i18n.t('msp:accessKey details') : i18n.t('established successfully')}
+          visible={keyDetailShow || modalShow}
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:263]: keyDetailShow  modalShow may be combined into one variable

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 2]
```diff
@@ -272,65 +280,35 @@ const Configuration = () => {
           <div className="rounded-sm p-4 container-key text-gray mb-4">
             <div className="flex items-center mb-2">
               <span>accessKey ID</span>
-              <span className="ml-32">{keyDetailInfo?.accessKey}</span>
+              <span className="ml-32">{keyDetailShow ? keyDetailInfo?.accessKey : createKeyInfo?.accessKey}</span>
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:283]: same as above

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 3]
```diff
@@ -272,65 +280,35 @@ const Configuration = () => {
...
             </div>
             <div className="flex items-center">
               <span>accessKey Secret</span>
-              <span className="ml-24">{keyDetailInfo?.secretKey}</span>
+              <span className="ml-24">{keyDetailShow ? keyDetailInfo?.secretKey : createKeyInfo?.secretKey}</span>
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:287]: same as above

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 4]
```diff
@@ -272,65 +280,35 @@ const Configuration = () => {
...
             </div>
           </div>
 
           <div className="flex items-center text-primary">
-            <div className="cursor-pointer" onClick={() => keyDetailInfo && downloadCsvFile(keyDetailInfo.id)}>
+            <div
+              className="cursor-pointer"
+              onClick={() => {
+                keyDetailShow
+                  ? keyDetailInfo && downloadCsvFile(keyDetailInfo.id)
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:296]: same as above

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 5]
```diff
@@ -272,65 +280,35 @@ const Configuration = () => {
...
-            <Button
-              key="close"
-              onClick={() =>
-                update({
-                  modalShow: false,
-                })
+            <Copy
+              selector=".container-key"
+              copyText={
+                keyDetailShow
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:307]: same as above

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 6]
```diff
@@ -251,17 +258,17 @@ const Configuration = () => {
...
           onCancel={() =>
             update({
-              keyDetailShow: false,
+              visible: false,
             })
           }
           width={720}
-          title={i18n.t('msp:accessKey details')}
-          visible={keyDetailShow}
+          title={visible ? i18n.t('msp:accessKey details') : i18n.t('established successfully')}
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:265]: logical error
`title={mode==='query' ? i18n.t('msp:accessKey details') : i18n.t('established successfully')}`?

File: shell/app/modules/msp/env-setting/configuration/index.tsx [hunk 7]
```diff
@@ -73,13 +73,13 @@ const Configuration = () => {
...
-  const [{ lang, currentPage, strategy, languages, modalShow, keyDetailShow }, updater, update] = useUpdate<IState>({
+  const [{ lang, currentPage, strategy, languages, mode, visible }, updater, update] = useUpdate<IState>({
     lang: '',
     strategy: '',
     languages: [],
-    modalShow: false,
-    keyDetailShow: false,
+    visible: false,
     currentPage: 1,
+    mode: 'create' || 'query',
```
Comment (Zero-Rock) [inline shell/app/modules/msp/env-setting/configuration/index.tsx:82]: redundant operator `||`

**Comment by daskyrk** [comment]
Update this pr title to make it easy to understand.

**Pull request merged**